.devcontainer/docker-compose.yml

```diff
@@ -1,14 +1,28 @@
 name: 'nextcloud-with-memories'
 services:
-  nextcloud:
+  nextcloud-server:
     build:
       context: .
       dockerfile: Dockerfile
     volumes:
+      # - nextcloud:/var/www/html
+      # - apps:/var/www/html/custom_apps
       - ./custom_apps:/var/www/html/custom_apps
-      - ../:/var/www/html/custom_apps/memories
+      - ./../:/var/www/html/custom_apps/memories

```

Comment:
Is the `./` at the start required? Seems a bit odd to me.

Reply:
Mmm, I think vscode formatter add them. I do not remember that I add it. But I do not remember now to be honest.